src/lib/components/Card/Card.stories.tsx

```diff
@@ -1,17 +1,12 @@
 import type { Meta, Story } from '@storybook/react/types-6-0';
+import Image from 'next/image';
 import type { CardProps } from './Card';
 import { Card } from './Card';
 
 export default {
   title: 'Components/Card',
   component: Card,
-  decorators: [
-    (Story): JSX.Element => (
-      <div className="h-1/2 w-1/2">
-        <Story />
-      </div>
-    ),
-  ],
+  decorators: [(Story): JSX.Element => <div className="h-1/2 w-1/2">{Story()}</div>],
```

Comment:
This change was important. This fixes the broken code previews in the storybook docs:
![Screenshot 2023-05-04 at 14 32 37](https://user-images.githubusercontent.com/1150767/236205370-a048e187-1a9d-4643-ac93-b8ecb629c3a2.png)